Changes

```diff
@@ -1,5 +1,7 @@
 {{$NEXT}}
 
+1.05 2021-11-12 
+    - Added alias names for Eswatini, Trinidad & Tobago, Côte D'Ivoire
```

Comment:
The cpan release will automatically add version and datetime
https://wikijs.deriv.cloud/en/Backend/guides/perl/cpan/cpan-maintenance#updating-the-changes-log
```suggestion
{{$NEXT}}
	- Added alias names for Eswatini, Trinidad & Tobago, Côte D'Ivoire
```